core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -287,14 +287,15 @@ THE SOFTWARE.
           btn = Event.element(o);
 
           <!-- trigger -->
-          new Ajax.Request(btn.getAttribute('url')+"/make"+(btn.checked?"Enable":"Disable")+"d", {
-            method: "POST",
-            onFailure : function(req,o) {
-                $('needRestart').innerHTML = req.responseText;
+          fetch(btn.getAttribute('url')+"/make"+(btn.checked?"Enable":"Disable")+"d", {
```

Comment:
While we're here, simplify

```suggestion
          fetch(btn.getAttribute('url')+"/make"+(btn.checked?"Enabled":"Disabled"), {
```

or save characters 😎 :trollface: 

```suggestion
          fetch(btn.getAttribute('url')+"/make"+(btn.checked?"En":"Dis")+"abled", {
```

(Untested.)